packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/User.kt

```diff
@@ -68,6 +68,30 @@ public interface User {
      */
     public val identities: List<UserIdentity>
 
+    /**
+     * Returns the provider type used to log the user in
+     * If a user logs out the authentication provider persists.
+     */
+    public val provider: AuthenticationProvider
+
+    /**
+     * Returns the current access token for the user.
+     * If a user logs out the access token is removed.
```

Comment:
```suggestion
     * If a user logs out, an empty access token is returned.
```

Removed is an ambiguous word IMO, so rephrased a little.